<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLogin</class>
 <widget class="QDialog" name="DlgLogin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgLogin{
	background-color: rgb(255, 255, 255);
}
#label{
	color: rgb(57,171,251);
	font-size: 26px;
}
#label_2,  #label_7, #lbVersion{
	color: rgb(153,153,153);
}
#editRoomID,#editNickName{
	color: rgb(51,51,51);
}

#widget QPushButton, #widget QLineEdit{
	min-height:36px;
	max-height:36px;
	border-radius: 18px;
}

#widget QLineEdit{
	background: transparent;
	border: 1px solid rgb(211,211,211);
	padding-left: 18px;
	padding-right: 18px;
}


#btnEnterMeeting{
	background: rgb(57,171,251);
	color:white;
}
#btnEnterMeeting:hover,#btnEnterMeeting:pressed{
	background: rgb(5,117,197);
}

#btnCreateMeeting{
	border: 1px solid rgb(57,171,251);
	color:  rgb(57,171,251);
	background: transparent;
}

#btnCreateMeeting:hover,#btnCreateMeeting:pressed{
	background: rgb(57,171,251);
	color:white;
}
#btnEnterMeeting:disabled,#btnCreateMeeting:disabled{
	background: rgb(204,204,204);
	color:white;
	border: 0px;
}

#widget_2 #line1,#widget_2 #line2{
	background: rgb(204,204,204);
	max-height: 1px;
}

#btnLoginSet{
	color:rgb(57,171,251);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>API Demo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>创建一个新房间或输入房间号加入已有房间</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="editRoomID">
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="placeholderText">
         <string>请输入房间号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="editNickName">
        <property name="placeholderText">
         <string>请输入昵称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnEnterMeeting">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>进入房间</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnCreateMeeting">
        <property name="text">
         <string>创建房间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">#line, #line_2
{
color:rgb(181, 0, 0);
min-height: 10px;
max-height: 10px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="line1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>或者</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="line2">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="btnLoginSet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>登录设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbVersion">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
